crates/polars-ops/src/frame/pivot/mod.rs

```diff
@@ -157,12 +162,24 @@ where
         .into_iter()
         .map(|s| s.as_ref().to_string())
         .collect::<Vec<_>>();
-
+    let values = match values {
+        Some(v) => v
+            .into_iter()
+            .map(|s| s.as_ref().to_string())
+            .collect::<Vec<_>>(),
+        None => pivot_df
+            // No value columns provided, use remaining columns
+            .get_column_names()
+            .into_iter()
+            .map(|s| s.to_string())
+            .filter(|s| !(index.contains(s) | columns.contains(s)))
```

Comment:
We can probably calculate this directly using set operations.

Reply:
Hey @stinodego ... I implemented this with set logic, but then realized that the output column order will not be stable. My psuedo logic is:

```
set(df.columns) - set(index) - set(values)
```
In rust using a `PlHashSet`, we don't necessarily get out the same order that we put in. Is this acceptable behavior? It looks like there is no way to ensure a stable order in hashbrown.